BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+ExchangeRateService service = new ExchangeRateService();
+
+// REST API to get exchange rates based on input currencies
+app.MapPost("/", service.GetExchangeRates);
```

Comment:
Delete this line. Instead of this we will use `app.MapControllers();`. It will allow you to manage your API dynamically via controllers.

Reply:
Also add app.UseHttpsRedirection();
It will automatically redirects it to the corresponding Https URL. For example, a request to http://anyapi.com would be redirected to https://anyapi.com